enterprise/backend/src/metabase_enterprise/transforms/models/transform.clj

```diff
@@ -49,14 +49,16 @@
    :run_trigger mi/transform-keyword})
 
 (t2/define-before-insert :model/Transform
-  [{:keys [source] :as transform}]
-  (assoc transform :source_type (transforms.util/transform-source-type source)))
+  [{:keys [source target] :as transform}]
+  (assoc transform
+         :source_type (transforms.util/transform-source-type source)
+         :target_database_id (:database target)))
```

Comment:
This feels like too many characters.

Table has `db_id`, can we not have `target_db_id`? 😄